tools/prefetch_cache_gcsfuse/main.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Prefetches files into local disk cache from specified bucket
+//
+// Usage:
+//
+//     prefetch_cache_gcsfuse cache_dir bucket_name
+//
+// This will prefetch the cache file
+//
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+)
+
+func run(args []string) (err error) {
+	// Extract arguments.
+	if len(args) < 2 || len(args) > 4 {
+		err = fmt.Errorf("Usage: %s cache_dir bucket_name [prefix] [delimiter]", os.Args[0])
+		return
+	}
+
+	cacheDir := args[0]
+	bucketName := args[1]
+	var prefix string
+	var delimiter string
+
+	if len(args) > 2 {
+		prefix = args[2]
+	}
+
+	// TODO ezl delimiter is not quite supported yet
```

Comment:
We don't need to specify the delimiter in the args.

The only use case for the preloader is:
```
> preload_gcsfuse /tmp/cache/path/ gs://bucket-name/foo/
```

Reply:
Got it.  I've updated the usage as indicated in the comment:

`prefetch_cache_gcsfuse cache_dir bucket_name [prefix]`